docs/vector.rst

```diff
@@ -106,7 +106,7 @@ Methods
 
 .. function:: a.norm(eps = 0)
 
-    :parameter a: (Vector)
+    :parameter a: (ti.Vector or ti.Vector.field)
```

Comment:
Hi Xudong,

I don't think the input here can be the whole field (please see the screenshot as an example). 'a' still has to be each entry of the vector field so it's a vector, and the modification here is not needed right?

Similar suggestions for functions like 'dot' and 'cross' later.

![image](https://user-images.githubusercontent.com/48085697/88769836-82abca80-d1af-11ea-905b-1cd6458d6f9f.png)

```suggestion
    :parameter a: (ti.Vector)
```

Reply:
Thanks for this reminder! It's fully my fault, I will test these functions tonight and give a feedback :D

Reply:
Sure, I have tested on my laptop. But it's always good to make sure.

Actually, I am wondering if these operations can actually be done field-wisely in the future, which sounds convenient. I.E. normalized the whole vector field, or get the norm of the whole (force) field. What would you think? @archibate